### PR TITLE
Improved documentation and fix container network

### PR DESCRIPTION
**Pull request by leandropineda**
## Changes

- Removed python sample orders and instant action and replaced it by bash snippets
- Fixed MQTT hostname on configurations and removed docker `host` mode
- Reduced sleep time on catch-all fake action

## Demo

[Peek 2022-10-19 16-23.webm](https://user-images.githubusercontent.com/8092356/196785516-e7ff009c-0b34-4de3-b644-217441a0453a.webm)

### CODE REVIEW

File: docker/deployment/docker-compose.yml [hunk 1]
```diff
@@ -47,9 +46,9 @@ services:
...
             - capabilities: [gpu]
     command: sleep infinity
-    network_mode: "host"
 
   mosquitto:
     image: eclipse-mosquitto
     command: mosquitto -c /mosquitto-no-auth.conf
-    network_mode: "host"
+    ports:
+      - "18883:1883"
```
Comment (miguelgarcia) [inline docker/deployment/docker-compose.yml:54]: is the extra `8` intended @leandropineda ?

Comment (leandropineda) [reply docker/deployment/docker-compose.yml:54]: Yes, just to avoid issues with a broker running on the default port.

**Review by FlorGrosso — APPROVED** [review]

File: vda5050_tb3_adapter/vda5050_tb3_adapter/tb3_adapter.py [hunk 2]
```diff
@@ -239,7 +239,7 @@ def process_vda_action_callback(self, goal_handle):
             ]:
                 feedback_msg.current_action.action_status = action_status
                 goal_handle.publish_feedback(feedback_msg)
-                time.sleep(5)
+                time.sleep(3)
```
Comment (FlorGrosso) [inline vda5050_tb3_adapter/vda5050_tb3_adapter/tb3_adapter.py:242]: Is there a reason for this update? Maybe we can leave it as a comment?

Comment (leandropineda) [reply vda5050_tb3_adapter/vda5050_tb3_adapter/tb3_adapter.py:242]: I just wanted to shorten a little bit the time it takes for this dummy action to execute.

**Pull request merged**